test/postman/dbp-test-collection.json

```diff
@@ -97,7 +97,8 @@
 							"raw": ""
 						},
 						"url": {
-							"raw": "{{acsUrl}}/api/discovery",
+							"raw": "https://{{acsUrl}}/api/discovery",
+							"protocol": "https",
```

Comment:
I presume all these substitutions of `{{acsUrl}}` to `https://{{acsUrl}}` (with "protocol" property set to "https") were done automatically by Postman? If so, it's a shame, this forces us to go down the https-only route (when in this place we wanted to have the flexibility to cater for http also and pass the full url, e.g. including the protocol, as `{{acsUrl}}`.

Reply:
BTW this probably isn't such a big deal, we should prioritise https anyway.